### PR TITLE
Start to dockerize 🐳 

### PR DESCRIPTION
**Pull request by jdtoombs**
It's late but started to Dockerize, may have to tweak some things and adjust as we go.

Will add docs later.

From the root directory run:
`docker-compose build` to build the container OR
`docker-compose up` (pretty sure this will build for you if you haven't already) 

Will run as normal, just now in a container 🐳

### CODE REVIEW

**Comment by jdtoombs** [comment]
I'll clean up and add documentation tomorrow

**Comment by jdtoombs** [comment]
Also - code owners should now be automatically requested for review when creating a PR and we now need one approval in order to merge. Eventually we can implement something to check our tests before allowing us to merge.

File: Dockerfile
Comment (nollidnosnhoj) [inline Dockerfile]: I think this is okay for now. But when we start deploying this, I think we should consider using node:14-alpine. It's a much smaller version of the node image which would be good. However, because it's a smaller image, it does take away some libraries. You can read more about it here https://hub.docker.com/_/node but I do not think we necessarily need it. But node:14 is fine for now.

Comment (jdtoombs) [reply Dockerfile]: Might as well switch it now

**Review by nollidnosnhoj — APPROVED** [review]

**Pull request merged**
